test/Common.h

```diff
@@ -102,6 +107,13 @@ bool isValidSemanticTestPath(boost::filesystem::path const& _testPath);
 
 bool loadVMs(CommonOptions const& _options);
 
+///Helper function to print the value of settings used (CommonOptions struct)
+///@p _filter should be the sequence of option names to be printed.
+void printOptions(std::ostream& _stream, std::string const& _linePrefix, CommonOptions const& _options, std::vector<std::string> const& _filter);
```

Comment:
Maybe `_selectedOptions` would be more self-explanatory than just `_filter`?
```suggestion
void printOptions(std::ostream& _stream, std::string const& _linePrefix, CommonOptions const& _options, std::vector<std::string> const& _selectedOptions);
```